src/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+const Footer = () => {
+    return (
+    <footer className = "footer pt-24 pb-3 flex flex-col text-white fixed bottom-0 left-0 w-screen md:flex-col-reverse md:justify-between md:flex-wrap-reverse">
+        <div className = "socialLinks md:order-2 md:basis-[30%] md:justify-end mx-10 mt-10 flex">
```

Comment:
Please ensure that the bottom of the left and right content is aligned! It also looks like the horizontal margin should be a bit larger, and the icons should maybe be a little smaller.

Current:
<img width="1161" alt="image" src="https://github.com/webimpactuw/poke-bay/assets/25410985/04d1877b-4ad5-47f1-9ec8-32bf5590081e">

Intended:
<img width="817" alt="image" src="https://github.com/webimpactuw/poke-bay/assets/25410985/de78da8f-b3f0-4648-9195-29b73d9a89f3">